src/FS.Embedded/Manifest/ManifestDirectoryInfo.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Reflection;
+
+namespace Microsoft.Extensions.FileProviders.Embedded.Manifest
+{
+    internal class ManifestDirectoryInfo : IFileInfo
+    {
+        private DateTimeOffset? _lastModified = null;
+
+        public ManifestDirectoryInfo(Assembly assembly, ManifestDirectory directory)
+        {
+            if (assembly == null)
+            {
+                throw new ArgumentNullException(nameof(assembly));
+            }
+
+            if (directory == null)
+            {
+                throw new ArgumentNullException(nameof(directory));
+            }
+
+            Assembly = assembly;
+            Directory = directory;
+        }
+
+        public bool Exists => true;
+
+        public long Length => -1;
+
+        public string PhysicalPath => null;
```

Comment:
How is this set?

Reply:
It is never set. See https://github.com/aspnet/FileSystem/blob/dev/src/FS.Abstractions/IFileInfo.cs#L25